prow/plugins/approve/approve.go

```diff
@@ -109,11 +115,11 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		Config: approveConfig,
 	}
 	pluginHelp.AddCommand(pluginhelp.Command{
-		Usage:       "/(approve|lgtm) [no-issue|cancel]",
+		Usage:       "/approve [no-issue|cancel]",
```

Comment:
Is it possible to have usage update based on the plugin's config? Beyond whether `lgtm` should be included, `no-issue` is also config driven

Reply:
Let me tinker and see if I can get that working.